### PR TITLE
feat(css/ast): use simple block for `@keyframes`

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Feat - use `SimpleBlock` for `@keyframes`

**BREAKING CHANGE:**

yes

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_codegen/src/lib.rs [hunk 1]
```diff
@@ -934,6 +926,7 @@ where
             ComponentValue::Value(n) => emit!(self, n),
             ComponentValue::DeclarationBlockItem(n) => emit!(self, n),
             ComponentValue::Rule(n) => emit!(self, n),
+            ComponentValue::KeyframeBlock(n) => emit!(self, n),
```
Comment (alexander-akait) [inline crates/swc_css_codegen/src/lib.rs:929]: After this we have `SimpleBlock` codegen logic in one place :star:

**Comment by alexander-akait** [comment]
@kdy1 Rebased, ready for review

**Review by kdy1 — APPROVED** [review]
Thanks!

**Review by kdy1 — APPROVED** [review]
Thanks!

---
swc-bump:
 - swc_css_ast --breaking

**Pull request merged**